sdk/cosmos/azure-cosmos-kafka-connect/src/main/java/com/azure/cosmos/kafka/connect/implementation/sink/KafkaCosmosBulkWriter.java

```diff
@@ -129,6 +134,8 @@ private CosmosBulkExecutionOptions getBulkExecutionOperations() {
                 .setMaxConcurrentCosmosPartitions(bulkExecutionOptions, this.writeConfig.getBulkMaxConcurrentCosmosPartitions());
         }
 
+        CosmosThroughputControlHelper.tryPopulateThroughputControlGroupName(bulkExecutionOptions, this.throughputControlConfig);
```

Comment:
Same for this one, the methods are in the form of `try....()` which indicates they are going to try to do something, and if they can't what's the behavior we are expecting?

Reply:
hmm, the tryxxx here is based on whether throughput control is enabled -> if it is enabled, then will apply. If not, then skip